tornjak-frontend/src/components/agent-create-join-token.tsx

```diff
@@ -0,0 +1,240 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import axios from 'axios';
+import GetApiServerUri from './helpers';
+import IsManager from './is_manager';
+import {
+  serverSelectedFunc
+} from 'redux/actions';
+import { RootState } from 'redux/reducers';
+// import PropTypes from "prop-types";
```

Comment:
nit: remove

Reply:
Needed for testing will be removed on last pr